projects/clickhouse/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM gcr.io/oss-fuzz-base/base-builder
```

Comment:
I think this file and build.sh need license headers.

Reply:
This will result in a travis failure.

Reply:
Thanks, fixed.